src/utils/db.js

```diff
@@ -0,0 +1,16 @@
+const { Sequelize } = require('sequelize');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const sequelize = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  dialect: 'postgres',
+});
```

Comment:
Consider adding a `port` option to the Sequelize configuration if your database is running on a non-default port. This ensures that the connection is made to the correct port.